### DISCUSSION TITLE
It is expected that the configuration item supports not reading classes from sub-packages.

### DISCUSSION THREAD

**Discussion by Typhus-xxj** [TongchengOpenSource/smart-doc#694] [Ideas]
The current configuration item: packageFilters, supports regular expressions. However, it cannot effectively exclude sub-packages under a certain package.
目前配置项：packageFilters，支持正则表达式。但是并不能有效排除某个包下的子包
<img width="280" alt="image" src="https://user-images.githubusercontent.com/48980471/230337287-f1fa792f-5ccc-4bce-8e64-cc28c3f9c4e5.png">

Currently, I want to generate the content under the Controller package normally without generating the content of sub-packages.
目前想正常生成Controller包下的内容，不生成子包的内容。

**Comment by shalousun** [comment]
@Typhus-xxj Pull requests can be submitted for collaborative contributions.

**Comment by Typhus-xxj** [comment]
ok

**Comment by chenqi146** [comment]
This item might be better implemented with Ant Path.
此项或许ant path是比较好的实现方案
